chipsec/defines.py

```diff
@@ -139,67 +139,99 @@ def scan_single_bit_mask(bit_mask):
 COMPRESSION_TYPE_NONE = 0
 COMPRESSION_TYPE_TIANO = 1
 COMPRESSION_TYPE_UEFI = 2
-COMPRESSION_TYPE_LZMA  = 3
+COMPRESSION_TYPE_LZMA = 3
 COMPRESSION_TYPE_BROTLI = 4
 COMPRESSION_TYPE_EFI_STANDARD = 5
 COMPRESSION_TYPE_UNKNOWN = 6
-COMPRESSION_TYPES_ALGORITHMS = [COMPRESSION_TYPE_LZMA, COMPRESSION_TYPE_TIANO, COMPRESSION_TYPE_UEFI, COMPRESSION_TYPE_BROTLI, COMPRESSION_TYPE_NONE]
-COMPRESSION_TYPES = [COMPRESSION_TYPE_NONE, COMPRESSION_TYPE_TIANO, COMPRESSION_TYPE_UEFI, COMPRESSION_TYPE_LZMA, COMPRESSION_TYPE_BROTLI, COMPRESSION_TYPE_EFI_STANDARD, COMPRESSION_TYPE_UNKNOWN]
+COMPRESSION_TYPES_ALGORITHMS = [
+    COMPRESSION_TYPE_LZMA,
+    COMPRESSION_TYPE_TIANO,
+    COMPRESSION_TYPE_UEFI,
+    COMPRESSION_TYPE_BROTLI,
+    COMPRESSION_TYPE_NONE,
+]
+COMPRESSION_TYPES = [
+    COMPRESSION_TYPE_NONE,
+    COMPRESSION_TYPE_TIANO,
+    COMPRESSION_TYPE_UEFI,
+    COMPRESSION_TYPE_LZMA,
+    COMPRESSION_TYPE_BROTLI,
+    COMPRESSION_TYPE_EFI_STANDARD,
+    COMPRESSION_TYPE_UNKNOWN,
+]
+
 
 def DB(val):
-    return struct.pack('<B', val)
+    return struct.pack("<B", val)
+
+
 def DW(val):
-    return struct.pack('<H', val)
+    return struct.pack("<H", val)
+
+
 def DD(val):
-    return struct.pack('<L', val)
+    return struct.pack("<L", val)
+
+
 def DQ(val):
-    return struct.pack('<Q', val)
+    return struct.pack("<Q", val)
+
+
+SIZE2FORMAT = {1: "B", 2: "H", 4: "I", 8: "Q"}
 
-SIZE2FORMAT = {
-    1: 'B',
-    2: 'H',
-    4: 'I',
-    8: 'Q'
-}
 
 def bytestostring(mbytes):
     if isinstance(mbytes, bytes):
         return mbytes.decode("latin_1")
     else:
         return mbytes
 
+
 def stringtobytes(mstr):
     if isinstance(mstr, str):
         return mstr.encode("latin_1")
     else:
         return mstr
 
+
 def pack1(value, size):
     """Shortcut to pack a single value into a string based on its size."""
     return struct.pack(SIZE2FORMAT[size], value)
 
+
 def unpack1(string, size):
     """Shortcut to unpack a single value from a string based on its size."""
     return struct.unpack(SIZE2FORMAT[size], string)[0]
 
+
 def get_version():
     version_strs = []
     chipsec_folder = os.path.abspath(chipsec.file.get_main_dir())
-    for fname in sorted([x for x in os.listdir(os.path.join(chipsec_folder, "chipsec")) if x.startswith('VERSION')]):
+    for fname in sorted(
+        [
+            x
+            for x in os.listdir(os.path.join(chipsec_folder, "chipsec"))
+            if x.startswith("VERSION")
+        ]
+    ):
```

Comment:
This level of formatting seems a bit excessive for comprehensions.  However, it is readable and will just take some time to get used to.

Reply:
Agreed. See my other comment about customization.